bigtable-hbase-1.x-parent/bigtable-hbase-1.x/src/test/java/com/google/cloud/bigtable/hbase1_x/TestMetrics.java

```diff
@@ -125,35 +124,31 @@ public void tearDown() throws IOException {
   public void readRows() throws IOException, InterruptedException {
     Table table = connection.getTable(TABLE_NAME);
 
-    Stopwatch readRowsStopwatch = Stopwatch.createStarted();
+    Stopwatch stopwatch = Stopwatch.createStarted();
     Result result = table.get(new Get(new byte[2]));
-    long readRowsTime = readRowsStopwatch.elapsed(TimeUnit.MILLISECONDS);
+    long methodInvocationLatency = stopwatch.elapsed(TimeUnit.MILLISECONDS);
 
     fakeDataService.popLastRequest();
 
-    AtomicLong readRowFailure =
-        fakeMetricRegistry.results.get(
-            "google-cloud-bigtable.grpc.method.ReadRow.retries.performed");
-    AtomicLong operationLatency =
-        fakeMetricRegistry.results.get(
-            "google-cloud-bigtable.grpc.method.ReadRow.operation.latency");
-    AtomicLong tableGetLatency =
-        fakeMetricRegistry.results.get("google-cloud-bigtable.table.get.latency");
-
-    Assert.assertEquals(3, readRowFailure.get());
-
-    long operationLatencyResult = operationLatency.get();
-    Assert.assertTrue(
-        "operation latency for ReadRow took longer than expected, readRowsTime: "
-            + readRowsTime
-            + ", operationLatency: "
-            + operationLatencyResult
-            + " difference: "
-            + Math.abs(readRowsTime - operationLatencyResult),
-        operationLatencyResult <= readRowsTime + 50 && operationLatencyResult >= readRowsTime - 50);
-    Assert.assertTrue(
-        "operation latency for table.get took longer than expected",
-        tableGetLatency.get() <= readRowsTime + 50 && operationLatencyResult >= readRowsTime - 50);
+    long readRowRetriesPerformed =
```

Comment:
I dont think you need readRows prefixes as the test name already implies that context

Reply:
Also, might be goods to suffix these with Metric. ie `retriesPerformedMetric`